_layouts/use.html

```diff
@@ -21,6 +21,11 @@
         into the mods folder.
     </strong>
 </p>
+<p>
+    It is recommended that you read the
+    <a href="https://fabricmc.net/wiki/faq:user">Frequently Asked Questions for users!</a>
+    Modding-friendly alternative launchers such as MultiMC are recommended over Minecraft Launcher. 
```

Comment:
I don't especially want to push 3rd party launchers this hard, the offiical and reccomended way to install fabric using the offical launcher. Its the only launcher that can be fully trusted with users creds, only launcher I can easily provide support for/guarantee it works as expected. Most players are also used to the official launcher installing mods can be daunting so keeping change to a minimum is best. 

There is nothing wrong with the other launchers, I just dont feel comfortable recommending them over the official launcher.

Reply:
>the offiical and reccomended way to install fabric using the offical launcher

Ah, multiple pages of the Wiki, specifically the [FAQ](https://fabricmc.net/wiki/faq:user) and [Installation Instructions](https://fabricmc.net/wiki/install), say that MultiMC is recommended so I thought it'd be consistent to have that there too. 

Should that recommendation be removed from the Wiki too?

I completely understand your concerns. Removed it.

Reply:
Confirmed on Discord: MultiMC is no longer the recommended way for users of Fabric.